lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-implementation/src/main/java/eu/learnpad/cw/CWXwikiBridge.java

```diff
@@ -362,7 +362,7 @@ public Recommendations getRecommendations(String modelSetId, String artifactId,
 
         for (String userId : potentialUsers) {
             UserData user = new UserData();
-            user.id = userId;
+            user.id = this.removePrefixes(userId);
```

Comment:
If we want to remove XWiki notion from user IDs in the Platform, then this line is not needed anymore because SIM will have only `bbarnes` in events.